test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3109,6 +3109,58 @@ def setup_rnn_tests():
 
 setup_rnn_tests()
 
+# Unary floating ufunc tests
+# Tests that integral arguments are properly cast to float
+unary_floating_ufuncs = (
+    'cos',
```

Comment:
Reorder this list logically. For example, you can start with the trig functions, and put a few on the same line: `'cos', 'sin', 'tan'`...

Reply:
Agreed! I reordered the list (https://github.com/pytorch/pytorch/pull/33322/commits/174d85e75be51508760b0406ef21cccdd462e18d).

Here is the revised list:

```
unary_floating_ufuncs = (
    'sin', 'cos', 'tan',
    'asin', 'acos', 'atan',
    'sinh', 'cosh', 'tanh',
    'ceil', 'floor',
    'exp', 'expm1', 'log', 'log10', 'log1p', 'log2',
    'sqrt', 'rsqrt',
    'trunc',
    'erf', 'erfc', 'erfinv',
    'lgamma', 'digamma',
    'sigmoid'
)
```

Let me know if this requires further changes!

Reply:
I took the liberty of reordering list in `test_torch.py` as well (https://github.com/pytorch/pytorch/pull/33322/commits/0b0fdef7ec56ae1ebdc13bb93656d575065e9218). The order is similar as above for consistency.